src/abi.h

```diff
@@ -106,6 +106,7 @@ typedef enum {
     NETPLAN_AUTH_KEY_MANAGEMENT_NONE,
     NETPLAN_AUTH_KEY_MANAGEMENT_WPA_PSK,
     NETPLAN_AUTH_KEY_MANAGEMENT_WPA_EAP,
+    NETPLAN_AUTH_KEY_MANAGEMENT_WPA_SAE,
```

Comment:
ABI checker does not mention any issue here... But this might be changing the `NETPLAN_AUTH_KEY_MANAGEMENT_8021X` value implicitly.

Can we move the new `_WPA_SAE` value down below `_8201X`, this way only the `_MAX` value would be affected in the ABI.

Reply:
Done